sqlglot/dialects/postgres.py

```diff
@@ -327,8 +327,7 @@ class Tokenizer(tokens.Tokenizer):
             "<@": TokenType.LT_AT,
             "|/": TokenType.PIPE_SLASH,
             "||/": TokenType.DPIPE_SLASH,
-            "BEGIN": TokenType.COMMAND,
-            "BEGIN TRANSACTION": TokenType.BEGIN,
+            "BEGIN": TokenType.BEGIN,
```

Comment:
Are we sure the syntax [here](https://www.postgresql.org/docs/current/sql-begin.html) is properly supported or, if not, we emit a `Command`?